src/Xlsx/RowIterator.php

```diff
@@ -124,6 +124,9 @@ public function next()
                             $this->valueTransformer->transform($this->xml->readString(), $type, $style)
                         );
                         break;
+                    case 'is':
+                        $rowBuilder->addValue( $columnIndex, $this->xml->readString() );
+                        break;
```

Comment:
Could you please explain a little more in the PR description the problem and your solution ?